Update README.md to describe Oniro

jrico-eclipse opened the pull request.

jrico-eclipse committed:
Update README.md to describe Oniro

jrico-eclipse commented:
![archOniro](https://github.com/user-attachments/assets/c9b09af5-750c-4fa5-918a-8959022daad2)
this picture needs to be added

jarmark commented on profile/README.md:
```diff
@@ -1,9 +1,40 @@
-The Eclipse Oniro for OpenHarmony organization hosts the code and repositories for add-ons and enhancements to the [OpenHarmony](https://www.openharmony.cn) project. These enhancements are part of the broader [Oniro Project](https://oniroproject.org/), an open-source, vendor-neutral operating system managed by the Eclipse Foundation.
-
-## Objective of the Project
+**Oniro** is an Eclipse Foundation Project dedicated to the development of an open source vendor-neutral Operating System (OS) platform. The Oniro Project was established through a collaboration between two global open source foundations: The Eclipse Foundation and The OpenAtom Foundation. Leveraging the solid foundation of *OpenHarmony*, an open source project operated by the OpenAtom Foundation ...
```
I'd use the phrase "solid grounds" or something similar to avoid using the word "foundation" in the meaning "of a base" right after talking about 2 Foundations.

jarmark commented on profile/README.md:
```diff
@@ -1,9 +1,40 @@
-## Objective of the Project
+**Oniro** is an Eclipse Foundation Project dedicated to the development of an open source vendor-neutral Operating System (OS) platform. The Oniro Project was established through a collaboration between two global open source foundations: The Eclipse Foundation and The OpenAtom Foundation. Leveraging the solid foundation of *OpenHarmony*, an open source project operated by the OpenAtom Foundation ...
 
 The goal of this project is to build upon OpenHarmony, extending it with additional functionalities tailored for global markets. This collaboration between the Eclipse Foundation and the OpenAtom Foundation is aimed at driving the development and adoption of OpenHarmony on a global scale.
```
I suggest the following update: "This project aims to build a global ecosystem of connected devices. It extends OpenHarmony platform with functionalities tailored for global markets. The collaboration between the Eclipse Foundation and the OpenAtom Foundation is aimed at driving the development and adoption of OpenHarmony on a global scale."

jarmark reviewed (commented):
I suggest updates as commented.

Commit:
Adjust "What You’ll Find Here" section and add image

- Adjust "What You’ll Find Here" section to reflect what is available in
the repositories.
- Add missing image
- apply some other minor suggested changes

Signed-off-by: Francesco Pham <francesco.pham1@h-partners.com>
